requirements.txt

```diff
@@ -1,10 +1,10 @@
-addict==2.4.0
```

Comment:
I had to switch to `>=` instead of `==` because when we run the command `mamba env create -f ...` at the end it uses the `setup.py` that relies on `requirements.txt` and it was reinstalling the older versions. Let me know if that is OK, or if you think that there is a better approach.